mindsdb/interfaces/skills/skill_tool.py

```diff
@@ -84,6 +84,7 @@ def _make_retrieval_tools(self, skill: db.Skills) -> dict:
         params = skill.params
         return dict(
             name=params.get('name', skill.name),
+            source=params.get('knowledge_base', None),
```

Comment:
'knowledge_base' throws an error in sql UI, let's use 'source' instead. See my [PR](https://github.com/mindsdb/mindsdb/pull/9101)


> SQL statement cannot be parsed by mindsdb_sql - CREATE SKILL retrieval_skill using type=‘retrieval’, name=‘ANI_networks_product_wiki’, description=‘info from a wiki from a company called ANI networks, use it to answer all questions asked to this agent’, knowledge_base=“ani_networks_kb_bge”: Syntax error, unknown input: > name=‘ANI_networks_product_wiki’, > description=‘info from a wiki from a company called ANI networks, use it to answer all questions asked to this agent’, > knowledge_base=“ani_networks_kb_bge” --^^^^^^^^^^^^^^ Expected symbol: “[identifier]”